code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -13,7 +13,7 @@
 	species_language_holder = /datum/language_holder/felinid
 	disliked_food = GROSS | RAW | CLOTH
 	var/original_felinid = TRUE //set to false for felinids created by mass-purrbation
-	payday_modifier = 0.75
+	payday_modifier = 2 // Worthy pay for the master race
```

Comment:
```suggestion
	payday_modifier = -2 // Worthy pay for the master race
```

Reply:
This is secretly a massive stealth buff as this will underflow and give them effectively infinite credits

Reply:
```suggestion
	payday_modifier = 0 // Worthy pay for the master race
```